src/screens/dialogs/access-file-dialog.tsx

```diff
@@ -7,8 +7,13 @@ import {
   AuthContext
 } from "../../context/index";
 import toast from "react-hot-toast";
+import { unshareFileService } from "../../services/files/unshare-file.service";
 
-export const AccessManagementDialog = () => {
+interface UserWithAccessRowProps {
```

Comment:
This component shouldn't receive any prop. You're just using the `user` props in the `key` property of the `div` in line 86, which I think it's not necessary.